### DISCUSSION TITLE
2.16.* subtitleTextAttributes how to set before ios 15?

### DISCUSSION THREAD

**Discussion by daminik00** [LeoNatan/LNPopupController#550] [General] [answered]
Hello!

I am working with your fantastic library and currently trying to get a grasp on using the ﻿subtitleTextAttributes in the Appearance before the iOS 15.
Despite my best efforts and extensive reading, I'm having trouble getting it to work as I expected. I've tried to insert this feature into my project, but unfortunately, it hasn't been successful.
Could you kindly explain or provide an example of how to implement ﻿subtitleTextAttributes in the Appearance before iOS 15 using your library?
Your support on this matter would be greatly appreciated!
Specific details related to your advice would undoubtedly be a significant help for others who are in the same situation facing this challenge.
Thank you in advance!

Best Regards,
Daniiil

**Comment by LeoNatan** [comment, marked as answer]
Hello,
To set attributes before iOS 15, set the `__subtitleTextAttributes` property with a dictionary (as you would with an `NSAttributedString`). 
Cheers!

**Reply by daminik00** [reply]
Thank you!!!
